mqtt/mqttd/src/broker/bootstrap/edgehub.rs

```diff
@@ -126,6 +163,44 @@ async fn server_certificate_renewal(renew_at: DateTime<Utc>) {
     }
 }
 
+async fn start_sidecars(
+    broker_handle: BrokerHandle,
+    system_address: String,
+) -> (SidecarShutdownHandle, JoinHandle<Result<()>>) {
+    let (termination_handle, tx) = channel::<()>();
+
+    let event_loop = tokio::spawn(async move {
+        info!("starting command handler...");
+        let (mut command_handler_shutdown_handle, command_handler_join_handle) =
+            start_command_handler(broker_handle, system_address).await?;
+
+        tx.await?;
+
+        command_handler_shutdown_handle.shutdown().await?;
+        command_handler_join_handle.await??;
+        info!("command handler shutdown.");
+        Ok::<(), anyhow::Error>(())
+    });
+
+    (SidecarShutdownHandle(termination_handle), event_loop)
+}
```

Comment:
A few comments on this structure:
- This function is called by start_server() and is not publicly accessible. I chose to put it only in the edgehub bootstrap because currently we have no generic-feature-only sidecars.
- I have this nested shutdown handle structure because if someone wants to add a new sidecar, the won't have to add more code to the start_server() method.